sys-apps/util-linux/files/su-l.pamd

```diff
@@ -0,0 +1,5 @@
+auth	   include      su
+account    include      su
+password   include      su
+session	   optional pam_lastlog.so
+session	   include      su
```

Comment:
I assume this doesn't require util-linux's su binary

Reply:
I'll try to test it manually and see if something is off

Reply:
`su -l -c whoami - root` from user core and `su -l -c whoami - core` from user core both worked with the respective password set